docs/source/model_doc/albert.rst

```diff
@@ -50,7 +50,10 @@ AlbertTokenizer
 Albert specific outputs
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-.. autoclass:: transformers.modeling_albert.AlbertForPretrainingOutput
+.. autoclass:: transformers.modeling_albert.AlbertForPreTrainingOutput
```

Comment:
Had a typo with all model output types using `Pretraining` instead of `PreTraining`. This changes 4 names (but those aren't directly accessible so this shouldn't break anything).